changelog/ambiguous-lambda.dd

```diff
@@ -0,0 +1,11 @@
+Using the ambiguous `(args) => {}` syntax now triggers a deprecation message
+
+Newcomers from other languages with builtin delegates  (such as Javascript and C#)
+would often intuitively use `(args) => { /* body */ }` for delegate / function literal.
+
+However, D's delegate syntax uses either a simple arrow (`(args) => body` or `args => body`),
```

Comment:
I would start the paragraph by explaining that the mentioned syntax does not have the expected effect. For example:

"[...] would often use `(args) => { /* body */ }` for delegate / function literal.

However, in D, this syntax results in a delegate that returns a delegate without any other side effects. This may trigger hard-to-debug bugs, therefore it is now deprecated.

Since the D delegate syntax uses either a simple arrow (examples), or braces (examples), if a delegate returning a delegate is indeed the intended usage, use `(args) { return () => /* body */; }` or `(args) => () { /* body */ }` instead."

How does that sound?

Reply:
I actually cut the last sentence, hope it's more readable now.